lasagne/layers/cuda_convnet.py

```diff
@@ -198,19 +202,33 @@ def get_output_for(self, input, **kwargs):
 
 
 class MaxPool2DCCLayer(CCLayer):
-    def __init__(self, incoming, ds, ignore_border=False, strides=None,
+    def __init__(self, incoming, pool_size, ignore_border=False, stride=None,
                  dimshuffle=True, **kwargs):
         from pylearn2.sandbox.cuda_convnet.pool import MaxPool
 
+        if 'pad' in kwargs:
+            raise NotImplementedError("MaxPool2DCCLayer does not "
+                                      "support padding")
```

Comment:
Good idea, that's more descriptive than letting the base class constructor raise something about an unknown kwarg.